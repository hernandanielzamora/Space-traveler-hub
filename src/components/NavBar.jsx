import React from 'react';
import logo from '../media/logo.png';

const NavBar = () => (
  <>
    <nav className="nav-container flex">
      <div className="logo-container flex">
        <img src={logo} alt="Space Travelers Logo" className="navbar-logo" />
        <h1>Space Travelers Hub</h1>
      </div>
      <div className="links-list flex">
        <ul className="flex list">
          <li>Rockets</li>
          <li>Missions</li>
          <li>My Profile</li>
        </ul>
      </div>
    </nav>
  </>
);

export default NavBar;
