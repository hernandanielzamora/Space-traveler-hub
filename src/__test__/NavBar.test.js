import React from 'react';
import { render } from '@testing-library/react';
import { BrowserRouter as Router } from 'react-router-dom';
import NavBar from '../components/NavBar';

it('renders Component correctly', () => {
  const tree = render(
    <Router>
      <NavBar />
    </Router>,
  );
  expect(tree).toMatchSnapshot();
});
