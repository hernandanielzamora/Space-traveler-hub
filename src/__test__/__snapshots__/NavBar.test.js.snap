// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders Component correctly 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <nav
        class="nav-container flex"
      >
        <div
          class="logo-container flex"
        >
          <img
            alt="Space Travelers Logo"
            class="navbar-logo"
            src="logo.png"
          />
          <h1>
            Space Travelers Hub
          </h1>
        </div>
        <div
          class="links-list flex"
        >
          <ul
            class="flex list"
          >
            <li
              class="navigation-link"
            >
              <a
                activeclassname="active-link"
                aria-current="page"
                class="list-link active"
                href="/"
              >
                Rockets
              </a>
            </li>
            <li
              class="navigation-link"
            >
              <a
                activeclassname="active-link"
                class="list-link"
                href="/missions"
              >
                Missions
              </a>
            </li>
            <li
              class="navigation-link"
            >
              <a
                activeclassname="active-link"
                class="list-link"
                href="/profile"
              >
                Profile
              </a>
            </li>
          </ul>
        </div>
      </nav>
    </div>
  </body>,
  "container": <div>
    <nav
      class="nav-container flex"
    >
      <div
        class="logo-container flex"
      >
        <img
          alt="Space Travelers Logo"
          class="navbar-logo"
          src="logo.png"
        />
        <h1>
          Space Travelers Hub
        </h1>
      </div>
      <div
        class="links-list flex"
      >
        <ul
          class="flex list"
        >
          <li
            class="navigation-link"
          >
            <a
              activeclassname="active-link"
              aria-current="page"
              class="list-link active"
              href="/"
            >
              Rockets
            </a>
          </li>
          <li
            class="navigation-link"
          >
            <a
              activeclassname="active-link"
              class="list-link"
              href="/missions"
            >
              Missions
            </a>
          </li>
          <li
            class="navigation-link"
          >
            <a
              activeclassname="active-link"
              class="list-link"
              href="/profile"
            >
              Profile
            </a>
          </li>
        </ul>
      </div>
    </nav>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
